schema-registry/serdes/src/main/java/com/hortonworks/registries/schemaregistry/serdes/avro/kafka/KafkaAvroSerializer.java

```diff
@@ -37,27 +40,35 @@
     public static final String SCHEMA_GROUP = "schema.group";
     public static final String SCHEMA_NAME_KEY_SUFFIX_ = "schema.name.key.suffix";
     public static final String SCHEMA_NAME_VALUE_SUFFIX_= "schema.name.value.suffix";
+    public static final String STORE_SCHEMA_IN_HEADER = "store.schema.in.header";
```

Comment:
you may want to rename `store.schema.in.header` to `store.schema-id.in.header` or any other value which describes the property well.